### PR TITLE
Adds support for looking through a user's photo library

### PR DESCRIPTION
**Pull request by orta**
End goal is this:
screen shot 2017-06-27 at 11 28 41

Not quite there yet as it just uses stubbed images inside the storybooks, I mainly wanted a PR so I could review the changes.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/components/consignments/components/__tests__/ImageSelection-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Consignments/Components/ArtistSearchResults.tsx [hunk 1]
```diff
@@ -63,24 +62,39 @@ const noResults = props => {
...
-        onChangeText: props.onChangeText,
-        autoFocus: typeof jest === "undefined" /* TODO: https://github.com/facebook/jest/issues/3707 */,
-      }}
-      style={{ flex: 0 }}
-    />
+const render = (props: ArtistQueryData) => {
+  const rowForResult = result =>
+    <Result key={result.id} onPress={() => props.resultSelected(result)}>
+      <ResultContainers>
+        {result.image ? <Image source={{ uri: result.image.url }} /> : null}
```
Comment (mzikherman) [inline src/lib/Components/Consignments/Components/ArtistSearchResults.tsx]: `result.image && <Image ..... ./>` maybes?

Comment (orta) [inline src/lib/Components/Consignments/Components/ArtistSearchResults.tsx]: Yes!

File: src/lib/components/consignments/components/ConsignmentBG.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components/native"
```
Comment (orta) [inline src/lib/components/consignments/components/ConsignmentBG.tsx]: boo ^ lowercases

**Comment by orta** [comment]
Finallllly

**Comment by orta** [comment]
Today this doesn't look pretty, but I want to get it in so I can iterate.

screen shot 2017-07-06 at 14 19 56

**Comment by orta** [comment]
This also adds bundler, and locks the `psych` gem so that our lockfile is determinate.

**Comment by orta** [comment]
PR's still alive, gonna re-assign

**Comment by l2succes** [comment]
The code looks good but since the user photo library view is not accessible, I couldn't test on device yet.

Feel free to merge after resolving the conflicts.

File: Example/Gemfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+source "https://rubygems.org"
+
+gem "cocoapods"
+gem "psych"
```
Comment (orta) [inline Example/Gemfile]: screen shot 2017-07-13 at 12 52 00

**Pull request merged**